cmd/config.go

```diff
@@ -51,6 +53,40 @@ func (c *NukeConfig) InBlacklist(searchID string) bool {
 	return false
 }
 
+func (c *NukeConfig) ValidateAccount(accountID string, aliases []string) (*NukeConfigAccount, error) {
```

Comment:
Imho, I liked the old signature of `ValidateAccount`, where it only validates and returns error(s).

Reply:
That's very true. Fixed.